### DISCUSSION TITLE
How to setup DB connection values from a different secret?

### DISCUSSION THREAD

**Discussion by karanbalani** [apache/superset#24777] [Q&A / Help]
So I have a custom secret created in my EKS cluster, I want the values from that secret to be used in `values.yaml` file.
How can I achieve that?

Currently, If I'm trying to set up an `env` in my `values.yaml` file like this:
<img width="346" alt="image" src="https://github.com/apache/superset/assets/29383381/fd63356c-7b67-4e57-8709-2b5d844be1fa">

And then trying to refer its value for setting the connection values like:
<img width="429" alt="image" src="https://github.com/apache/superset/assets/29383381/35c813bc-97ef-4126-82f6-bdc1901a95a2">

I get the value in the `superset-env` secret setup as:
<img width="365" alt="image" src="https://github.com/apache/superset/assets/29383381/c0178a26-3f13-42ea-bef3-56447c6a6c8a">

Technically, the values should be there. :(

Can anyone please help here?

**Comment by YoninL** [comment]
You can change the value of `envFromSecret` in https://github.com/apache/superset/blob/cacad56172225351f3c22a488aac839575f688dc/helm/superset/values.yaml#L54 to your own secret.
